v3/spring-cloud-aws-v3-autoconfigure/src/main/java/io/awspring/cloud/v3/autoconfigure/config/parameterstore/ParameterStoreConfigDataLoader.java

```diff
@@ -43,11 +37,10 @@
  */
 public class ParameterStoreConfigDataLoader implements ConfigDataLoader<ParameterStoreConfigDataResource> {
 
-	private final DeferredLogFactory logFactory;
-
 	public ParameterStoreConfigDataLoader(DeferredLogFactory logFactory) {
-		this.logFactory = logFactory;
-		reconfigureLoggers(logFactory);
+		BootstrapLoggingHelper.reconfigureLoggers(logFactory,
+				"io.awspring.cloud.v3.paramstore.ParameterStorePropertySource",
+				"io.awspring.cloud.v3.paramstore.ParameterStorePropertySource");
```

Comment:
there should be one entry of ParameterStorePropertySource.

Reply:
Ah, it should be actually different class. thx good spot.